tools/lsp/preview.rs

```diff
@@ -1437,6 +1437,43 @@ fn set_preview_factory(
     // Ensure that any popups are closed as they are related to the old factory
     i_slint_core::window::WindowInner::from_pub(ui.window()).close_all_popups();
 
+    compiled.set_debug_handler(
+        Rc::new(|location, text| {
+            let location = location.as_ref().and_then(|l| {
+                l.source_file.as_ref().map(|f| {
+                    let (line, column) = f.line_column(l.span.offset);
+
+                    (f.clone(), line, column)
+                })
+            });
+            if let Some((file, line, column)) = &location {
+                i_slint_core::debug_log!(
+                    "DEBUG {}:{}:{}> {text}",
```

Comment:
Not sure if "DEBUG" is needed there. 
I think i'd do something like

```suggestion
                    "{}:{line}:{column}: {text}",
```

What do other think ?